app/templates/components/login-form.hbs

```diff
@@ -9,6 +9,9 @@
   <div class="input-group">
     {{input name="password" id="password" placeholder="Password" type="password" value=password}}
   </div>
+  {{#link-to 'password.forgot' classNames="t-forgot-password"}}
```

Comment:
@JoshSmith this forgot-password link could use some styling when you get a chance.